shap/_explanation.py

```diff
@@ -378,19 +358,19 @@ def __getitem__(self, item) -> Explanation:
 
                         new_self = Explanation(
                             np.array(new_values),
-                            np.array(new_base_values),
-                            np.array(new_data),
-                            self.display_data,
-                            self.instance_names,
-                            np.array(new_data),
-                            t,  # output_names
-                            self.output_indexes,
-                            self.lower_bounds,
-                            self.upper_bounds,
-                            self.error_std,
-                            self.main_effects,
-                            self.hierarchical_values,
-                            self.clustering,
+                            base_values=np.array(new_base_values),
+                            data=np.array(new_data),
+                            display_data=self.display_data,
+                            instance_names=self.instance_names,
+                            feature_names=np.array(new_data),  # FIXME: this is probably a bug
```

Comment:
Here, this `feature_names=np.array(new_data)` line looks weird, hence flagged out as a potential bug.

Reply:
Good spot. A great example of why the new code using kwargs is much more readable than the previous style!

I agree it looks like a bug 🤔